knowledge-base/cannot-implicitly-convert-type-telerik.reporting.xxxreportsource-to-telerik.reportviewer.html5.webforms.reportsource-error.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer
+description: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer.
+type: how-to
+page_title: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer
+slug: cannot-implicitly-convert-type-telerik.reporting.xxxreportsource-to-telerik.reportviewer.html5.webforms.reportsource-error
+res_type: kb
+---
+
```

Comment:
There are many missing parts/sections of the agreed troubleshooting KB template - https://wiki.progress.com/display/DTS/Knowledge+Base+Content+Standards#KnowledgeBaseContentStandards-ComponentsofTroubleshootingArticles

For example - environment, steps to reproduce, cause, etc..